src/Apps/Artwork/Components/ArtworkRelatedArtists.tsx

```diff
@@ -24,16 +25,16 @@ export class ArtworkRelatedArtists extends React.Component<
         {({ user, mediator }) => {
           return (
             <Box mt={6}>
-              <Header title="Related artists" />
+              <Header
+                title="Related artists"
+                buttonHref={
+                  sd.APP_URL + "/artist/" + artist._id + "/related_artists"
```

Comment:
#minor can maybe just be `artist.id` (using slug instead of Mongo id).

Reply:
Looking at it - I should just use `artist.href`. Will update